.vscode/settings.json

```diff
@@ -1,3 +1,15 @@
 // Place your settings in this file to overwrite default and user settings.
```

Comment:
why would nobody get it? I might misunderstand the purpose of .gitignore, but I thought it prevents files from being checked in - not checked out